modulefiles/RadMonBuild.jet

```diff
@@ -0,0 +1,18 @@
+#%Module################################################
+# Modfule file for radmon
+########################################################
+export ver=2.0.2
+export FCOMP=mpiifort
+
+export CF=$FCOMP
+export FC=$FCOMP
```

Comment:
same question as above.

Reply:
As requested, I put the answer above.

Reply:
Same comment as above, please remove the modulefiles/OznMonBuild.* and modulefiles/RadMonBuild.* modulefiles. Thanks!

Reply:
Removed as requested.